src/Illuminate/Foundation/Console/VendorPublishCommand.php

```diff
@@ -89,9 +89,8 @@ protected function determineWhatShouldBePublished()
             return;
         }
 
-        [$this->provider, $this->tags] = [
```

Comment:
Simply reverting the breaking change. https://github.com/laravel/framework/commit/4f8041699b59c20277da8a9c96d669ed7550e493#commitcomment-21274372

If anyone want to chase this in a different way, feel free to make a PR and I close this one.